src/__experimental__/components/simple-color-picker/simple-color-picker.stories.js

```diff
@@ -44,10 +46,20 @@ function makeStory(storyName, themeSelector) {
     return (
       <State store={ store }>
         <SimpleColorPicker
-          availableColors={ availableColors }
           name={ name }
+          legend={ legend }
           onChange={ onChange }
-        />
+        >
+          {availableColors.map(color => (
+            <SimpleColor
+              color={ color }
+              key={ color }
+              aria-label={ color }
```

Comment:
Please, can you update the aria labels to the color name instead of the hex code.